src/azure-cli/azure/cli/command_modules/netappfiles/tests/latest/test_backup_commands.py

```diff
@@ -98,6 +116,9 @@ def test_list_backup(self):
 
         assert len(backup_list) == 1
 
+        if self.is_live or self.in_recording:
+            time.sleep(10)
```

Comment:
Could you share us the reason why we need `sleep` here? From my perspective, `az netappfiles account backup-policy create` in method `create_backup` is implemented as a synchronous command which will wait the status to `success` before it continues to execute next command. So you may not need sleep here.

Reply:
I was for some reason getting an error saying that the process above was still running and that I had to wait. I´ll remove this since the backup tests are skipped due to backup service not being stable enough.